stdlib/testing/influxql/series_agg_test.flux

```diff
@@ -0,0 +1,704 @@
+,,1,1970-01-05T20:00:00Z,m,1,0.3082540490684122
+,,1,1970-01-05T21:00:00Z,m,1,-0.2724359684571728
+,,1,1970-01-05T22:00:00Z,m,1,0.011784872578827432
+,,1,1970-01-05T23:00:00Z,m,1,0.19189080075398407
+,,1,1970-01-06T00:00:00Z,m,1,0.4010692090163104
+,,1,1970-01-06T01:00:00Z,m,1,-0.3974927591472375
+,,1,1970-01-06T02:00:00Z,m,1,0.5549536703053173
+,,1,1970-01-06T03:00:00Z,m,1,-0.060007745100103094
+,,1,1970-01-06T04:00:00Z,m,1,-0.818834026213009
+,,1,1970-01-06T05:00:00Z,m,1,0.30644732156601706
+,,1,1970-01-06T06:00:00Z,m,1,0.5466876889235377
+,,1,1970-01-06T07:00:00Z,m,1,-0.4487656554442424
+,,1,1970-01-06T08:00:00Z,m,1,0.06697200564618833
+,,1,1970-01-06T09:00:00Z,m,1,-0.019727583277435956
+,,1,1970-01-06T10:00:00Z,m,1,0.2785143808725318
+,,1,1970-01-06T11:00:00Z,m,1,-0.5721126504527568
+,,1,1970-01-06T12:00:00Z,m,1,-0.07797445220765423
+,,1,1970-01-06T13:00:00Z,m,1,0.5415176275149132
+,,1,1970-01-06T14:00:00Z,m,1,-0.1445515797425707
+,,1,1970-01-06T15:00:00Z,m,1,-0.2500652415619768
+,,1,1970-01-06T16:00:00Z,m,1,0.5608616065849172
+,,1,1970-01-06T17:00:00Z,m,1,-0.7306403933278911
+,,1,1970-01-06T18:00:00Z,m,1,0.054693268726029295
+,,1,1970-01-06T19:00:00Z,m,1,0.5980474067387233
+,,1,1970-01-06T20:00:00Z,m,1,0.14368604611073088
+,,1,1970-01-06T21:00:00Z,m,1,0.024831823459466107
+,,1,1970-01-06T22:00:00Z,m,1,-0.15899787074014582
+,,1,1970-01-06T23:00:00Z,m,1,-0.19778745612321147
+,,1,1970-01-07T00:00:00Z,m,1,0.2605104623709601
+,,1,1970-01-07T01:00:00Z,m,1,-0.24798456934381652
+,,1,1970-01-07T02:00:00Z,m,1,-0.02413169404236115
+,,1,1970-01-07T03:00:00Z,m,1,0.2850223923198476
+,,1,1970-01-07T04:00:00Z,m,1,-0.4533907284450288
+,,1,1970-01-07T05:00:00Z,m,1,-0.07631427012911585
+,,1,1970-01-07T06:00:00Z,m,1,0.06614318244779838
+,,1,1970-01-07T07:00:00Z,m,1,-0.34231345077450664
+,,1,1970-01-07T08:00:00Z,m,1,0.4749989855676307
+,,1,1970-01-07T09:00:00Z,m,1,-0.4400603107103018
+,,1,1970-01-07T10:00:00Z,m,1,0.08744582080929783
+,,1,1970-01-07T11:00:00Z,m,1,0.6911673740571932
+,,1,1970-01-07T12:00:00Z,m,1,-0.1416643419909407
+,,1,1970-01-07T13:00:00Z,m,1,-0.3020857474445764
+,,1,1970-01-07T14:00:00Z,m,1,0.13211207295926208
+,,1,1970-01-07T15:00:00Z,m,1,0.33436238731583623
+,,1,1970-01-07T16:00:00Z,m,1,0.08170793142825539
+,,1,1970-01-07T17:00:00Z,m,1,-0.4362527652716799
+,,1,1970-01-07T18:00:00Z,m,1,-0.5078189070762902
+,,1,1970-01-07T19:00:00Z,m,1,0.7027125984921005
+,,1,1970-01-07T20:00:00Z,m,1,0.11376676392370555
+,,1,1970-01-07T21:00:00Z,m,1,-0.4422745858695098
+,,1,1970-01-07T22:00:00Z,m,1,0.15894641183358554
+,,1,1970-01-07T23:00:00Z,m,1,0.16068490698806603
+"
+
+// SELECT difference(f) FROM m GROUP BY *
+t_difference = (tables=<-) => tables
+	|> range(start: influxql.minTime, stop: influxql.maxTime)
+	|> filter(fn: (r) => r._measurement == "m")
+	|> filter(fn: (r) => r._field == "f")
+	|> difference()
+	|> drop(columns: ["_start", "_stop", "_field"])
+	|> map(fn: (r) =>
+		({_time: r._time, difference: r._value}), mergeKey: true)
```

Comment:
`mergeKey` is deprecated and shouldn't be used. The transpiler should be updated to use `with` instead.

Reply:
In this case, I will change the transpiler to use the rename() function.
For GROUP BY *, I think I don't have a way to know the name of the tags to add to `with`?

Reply:
I think rename will end up being better anyway. Right now `rename` performance is the same as everything else, but it is also probably one of the easiest functions to make fast.